servers/quarkus-server/src/main/java/org/projectnessie/server/config/TieredVersionStoreConfig.java

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (C) 2020 Dremio
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.projectnessie.server.config;
+
+import static org.projectnessie.versioned.impl.TieredVersionStoreConfig.DEFAULT_COMMIT_RETRY_COUNT;
+import static org.projectnessie.versioned.impl.TieredVersionStoreConfig.DEFAULT_P2_COMMIT_RETRY_COUNT;
+
+import org.eclipse.microprofile.config.inject.ConfigProperty;
+
+import io.quarkus.arc.config.ConfigProperties;
+
+@ConfigProperties(prefix = "nessie.version.store.tiered")
```

Comment:
why not making this a super interface of each tiered store backend instead so that you could use different values for different implementations

Reply:
You mean like in the commit I've just pushed?

Reply:
Yes, and I guess that `ConfigProperties` is not needed anymore?

Reply:
yea, true, removed that